File: java-union-reader-writer-example/src/main/java/com/intellicess/union/java_union_reader_writer_example/client/model/JwlfResponseList.java
package com.intellicess.union.java_union_reader_writer_example.client.model;

import java.util.List;

public class JwlfResponseList {

    private List<JwlfResponse> list;

    private Long stableDataTimestamp;

    public List<JwlfResponse> getList() {
        return list;
    }

    public void setList(List<JwlfResponse> list) {
        this.list = list;
    }

    public Long getStableDataTimestamp() {
        return stableDataTimestamp;
    }

    public void setStableDataTimestamp(Long stableDataTimestamp) {
        this.stableDataTimestamp = stableDataTimestamp;
    }
}

File: csharp-union-reader-writer-example/csharp-union-reader-writer-example/JwlfResponseList.cs
using System;
using System.Collections.Generic;

namespace Union;
using Newtonsoft.Json;

public class JwlfResponseList
{

    [JsonProperty("list")]
    public List<JwlfResponse> List { get; set; }

    [JsonProperty("stableDataTimestamp")]
    public long StableDataTimestamp { get; set; }
}
